src/libraries/Core/Microsoft.Agents.Core/Serialization/ProtocolJsonSerializer.cs

```diff
@@ -76,6 +79,11 @@ public static void ApplyExtensionOptions(Func<JsonSerializerOptions, JsonSeriali
             }
         }
 
+        public static void AddEntityType(string entityTypeName, Type entityType)
```

Comment:
Why do we have this, if the backing dictionary is public?

Reply:
Good point.. Let me take it private.

Reply:
Cant take it private.. its static and used elsewhere.